src/components/editor/task-bar/connection-indicator.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { Dropdown } from 'react-bootstrap'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { UserAvatar } from '../../landing/layout/user-avatar/user-avatar'
+import './connection-indicator.scss'
+
+const ConnectionIndicator: React.FC = () => {
+  const userOnline = 1
+  return (
+    <Dropdown className="small">
```

Comment:
Please add a direction. Otherwise the dropdown menu will open to the right and break the site width

Reply:
done